recipes/jupyter-full-width/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "jupyter-full-width" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82bb5ec40652b7b482e8432e77707cfa05502711ed78ce6ea506fc93d52dd523
+
+build:
+  number: 0
+
```

Comment:
Add here the `script` build line: `script: python setup.py install --single-version-externally-managed --record record.txt`